packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -475,6 +475,14 @@ class TextStyle with Diagnosticable {
   /// The `package` argument must be non-null if the font family is defined in a
   /// package. It is combined with the `fontFamily` argument to set the
   /// [fontFamily] property.
+  ///
+  /// On Apple devices the strings 'CupertinoSystemText' and
+  /// 'CupertinoSystemDisplay' are used in [fontFamily] as proxies for the
+  /// Apple system fonts. They currently redirect to the equivilant of SF Pro
+  /// Text and SF Pro Display respectively. 'CupertinoSystemText' is designed
+  /// for fonts below 20 point size, and 'CupertinoSystemDisplay' is recommended
+  /// for sizes 20 and above. When used on non-Apple platforms, these strings
+  /// will fail and return the fallback font family as normal.
```

Comment:
"will fail" sounds like my app would crash? But fallback font sounds like it would just display another font?

Could we maybe say instead: When these strings are used on non-Apple platforms, the regular/normal fallback font family is used instead.